src/components/functional_module/src/plugin_manager.cc

```diff
@@ -310,6 +310,26 @@ void PluginManager::OnAppHMILevelChanged(
   }
 }
 
+bool PluginManager::CanAppChangeHMILevel(
+    application_manager::ApplicationSharedPtr app,
+    mobile_apis::HMILevel::eType new_level) {
+  DCHECK(app);
+  if (!app) {
+    return false;
+  }
+  bool result = true;
```

Comment:
If I understand correctly false only if one of plugins prohibits to change HMI level.
Write comment in code, please, it is not clear if the requirement is not front of eyes.